src/streaming/utils/CapabilitiesFilter.js

```diff
@@ -136,6 +136,22 @@ function CapabilitiesFilter() {
             }
             return supported
         });
+
+        // handle scte214:supplementalCodecs
+        as.Representation = as.Representation.map((rep) => {
+            const supplementalCodecs = rep['scte214:supplementalCodecs']
+            if (!supplementalCodecs) {
+                return rep
+            }
+            const codec = rep.mimeType + ';codecs="' + supplementalCodecs + '"';
+            const config = _createConfiguration(type, rep, codec);
+            const supported = capabilities.isCodecSupportedBasedOnTestedConfigurations(config, type);
```

Comment:
For this new configuration, `capabilities.runCodecSupportCheck` is never executed before. As a consequence `testedCodecConfigurations` in `Capabilities` should not contain this new configuration and the resulting variable `supported` should never be true. See also the description of the `isCodecSupportedBasedOnTestedConfigurations` function:

````
Checks whether a codec is supported according to the previously tested configurations.
Note that you need to call runCodecSupportCheck() first to populate the testedCodecConfigurations array.
This function only validates codec support based on previously tested configurations.
````